lib/aws_command.rb

```diff
@@ -11,7 +11,7 @@ class AwsCommand < Thor
 
       option :type, :required => true, :enum => LaunchHelper.get_aws_host_types,
              :desc => 'The host type of the new instances.'
-      option :env, :required => true, :aliases => '-e', :enum => SUPPORTED_ENVS,
+      option :env, :required => true, :aliases => '-e', :enum => SUPPORTED_ENVS << ENV['OO_CUSTOM_ENV'],
```

Comment:
In ruby all caps means const. So you're modifying a const here. I think it makes more sense to do this where the constant is being assigned in the first place on line 10 of this same file. Also, consider modifying the gce_command.rb script as well so that both have the same functionality.